airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -504,32 +507,31 @@ def stream_slices(
         self, sync_mode: SyncMode, cursor_field: Optional[List[str]] = None, stream_state: Optional[Mapping[str, Any]] = None
     ) -> Iterable[Optional[Mapping[str, Any]]]:
         parent_stream_state = None
-        if stream_state:
-            cursor_value = stream_state.get(self.cursor_field)
-            parent_stream_state = {self.parent.cursor_field: pendulum.parse(cursor_value).int_timestamp}
-        else:
-            cursor_value = self._start_date
-
+        cursor_value = (stream_state or {}).get(self.cursor_field, self._start_date)
+        if isinstance(cursor_value, str):
```

Comment:
totally fine if the answer is yes, but is there a need to support both strings and int? Does zendesk-support return both?

Reply:
Updated -- the default start date value would be a string formatted date time, updated to make the default value the start date converted to timestamp via pendulum.